composer.json

```
@@ -4,11 +4,11 @@
   "homepage": "https://www.myparcel.com",
   "type": "library",
   "require": {
-    "php": ">=7.1",
+    "php": "^7.1|^8.0",
     "ext-json": "*",
     "guzzlehttp/promises": "^1.0",
     "justinrainbow/json-schema": "^5.2",
-    "laravel/framework": "^5.4|^6.0|^7.0"
+    "laravel/framework": "^5.4|^6.0|^7.0|^8.0"
```

If we go the `>=` route above, it makes sense to do it here as well.
```suggestion
    "laravel/framework": ">=5.4"
```